autoload/neomake/makers/ft/haskell.vim

```diff
@@ -73,9 +73,12 @@ function! neomake#makers#ft#haskell#hdevtools() abort
     if !exists('s:uses_cabal')
         let s:uses_cabal = 0
         if executable('stack')
-            let rootdir = systemlist('stack --verbosity silent path --project-root')[0]
-            if empty(glob(rootdir . '/*.cabal'))
-                let s:uses_cabal = 1
+            let output = neomake#utils#systemlist(['stack', '--verbosity silent', 'path', '--project-root'])
+            if !empty(output)
+                let rootdir = output[0]
+                if empty(glob(rootdir . '/*.cabal'))
+                    let s:uses_cabal = 1
+                endif
```

Comment:
Ok, throwing up my hands for now...I'm struggling to keep hdevtools working in my environment, and I'm not convinced its worth it when ghc-mod works fine. I say, just invert the empty to !empty for now and I'll bang my head against this more when I can.